guardrails/cli/configure.py

```diff
@@ -32,7 +32,7 @@ def save_configuration_file(
 
     home = expanduser("~")
     guardrails_rc = os.path.join(home, ".guardrailsrc")
-    with open(guardrails_rc, "w") as rc_file:
+    with open(guardrails_rc, "w", encoding="utf-8") as rc_file:
```

Comment:
Wouldn't we want to open it as 'wt' for "write text"?

Reply:
I believe the "t" is default but would be good to me more explicit 👍🏼